docs/foundation/component-variables/README.md

```diff
@@ -174,10 +173,11 @@ The naming structure for component variables and mixins is as follows:
 
 <br>
 
-## Getting Started
+## Getting Started (Comment: "Resources" is a better section title here)
```

Comment:
:+1: i also think this section makes more sense at the top of the page, so that someone who just wants a link to the package can jump straight there

Reply:
There's a lot of work going on to restructure/reorganize content so that everything is in the right bucket and the content flow is smooth. The plan is to have Component Variables at the same level as Foundations and Components (and Layout). https://confluence.rei.com/display/TP/Cedar+Docs+-+Site+Map+-+Fall+19 
In that case, we can add Getting Started and Overview as sub menus (not tabs) under the main Component Variable menu item (like Layout).